birdman/model_base.py

```diff
@@ -257,40 +335,62 @@ def to_inference_object(
         if self.fit is None:
             raise ValueError("Model has not been fit!")
 
+        # if already Inference, just return
+        if isinstance(self.fit, az.InferenceData):
+            return self.fit
+        if isinstance(self.fit, list):
```

Comment:
zoops, spoke too soon.  Turns out that `self.fit` can be a tuple, so this will need to be 
```suggestion
        if isinstance(self.fit, list) or isinstance(self.fit, tuple):
```